data_diff/dbt_parser.py

```diff
@@ -11,6 +11,21 @@
 from dbt_artifacts_parser.parser import parse_run_results, parse_manifest
 from dbt.config.renderer import ProfileRenderer
 
+from data_diff.errors import (
```

Comment:
I would recommend to add a prefix `DataDiff` to that error names because when errors happen it will help to understand that they occur in data diff, not in a dbt package.

Reply:
updated